contracts/src/core/libraries/v3/address-directory/AddressDirectoryConstants.sol

```diff
@@ -31,4 +31,5 @@ library AddressDirectoryConstants {
 
     string internal constant CERT_VERIFIER_LEGACY_V1_NAME = "CERT_VERIFIER_LEGACY_V1";
     string internal constant CERT_VERIFIER_LEGACY_V2_NAME = "CERT_VERIFIER_LEGACY_V2";
+    string internal constant DISPERSER_REGISTRY_LEGACY_NAME = "DISPERSER_REGISTRY_LEGACY";
```

Comment:
Was this legacy disperser_registry even used by anyone? If not maybe we don't need to keep it around?